src/Css/Property/Property.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace TijsVerkoyen\CssToInlineStyles\Css\Property;
+
+class Property
+{
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var string
+     */
+    private $value;
+
+    /**
+     * Property constructor.
+     *
+     * @param string $name
+     * @param string $value
+     */
+    public function __construct($name, $value)
+    {
+        $this->name = $name;
+        $this->value = $value;
+    }
+
+    /**
+     * Get name
+     *
+     * @return string
+     */
+    public function getName()
+    {
+        return $this->name;
+    }
+
+    /**
+     * Get value
+     *
+     * @return string
+     */
+    public function getValue()
+    {
+        return $this->value;
+    }
+
+    /**
+     * Is this property important?
+     *
+     * @return bool
+     */
+    public function isImportant()
+    {
+        return (stristr($this->value, '!important') !== false);
```

Comment:
as you only need to check whether it is found, use ``stripos`` instead. It is faster and less memory intensive (this is even documented in the official doc: http://php.net/manual/en/function.strstr.php)